.github/workflows/release-please.yml

```diff
@@ -0,0 +1,12 @@
+on:
+  push:
+    branches:
+      - main
+name: release-please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v2
```

Comment:
i believe the correct name now is `google-github-actions/release-please-action`, i've added some of my CI failed because they changed it two week prior